### PR TITLE
Yoon

### PR DESCRIPTION
**Pull request by unow0517**
export wokring and dollar expand under construction

### CODE REVIEW

File: parsing.c [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+void	parse_tokens(t_shell *shell_info)
+{
+	t_command	*cmd_node;
+	t_token		*iterate;
+	int			len;
+
+	cmd_node = NULL;
+	iterate = shell_info->tokens;
+	len = number_of_tokens(shell_info);
```
Comment (tsimitop) [inline parsing.c:18]: remove line

File: pipe.c
Comment (tsimitop) [inline pipe.c:1]: file not needed

**Review by tsimitop — COMMENTED** [review]
change file name to tokenizer

**Review by tsimitop — COMMENTED** [review]
I reviewed the changes

File: parsing.c (hunk 1 above)
Comment (unow0517) [reply parsing.c:18]: yes

File: pipe.c
Comment (unow0517) [reply pipe.c:1]: file deleted, also makefile must be edited

**Pull request merged**